packages/@aws-cdk/aws-redshift/lib/private/database-query-provider/table.ts

```diff
@@ -40,7 +40,7 @@ async function createTable(
   tableAndClusterProps: TableAndClusterProps,
 ): Promise<string> {
   const tableName = tableNamePrefix + tableNameSuffix;
-  const tableColumnsString = tableColumns.map(column => `${column.name} ${column.dataType}`).join();
+  const tableColumnsString = tableColumns.map(column => `${column.name} ${column.dataType}${getEncodingColumnString(column)}`).join();
```

Comment:
```suggestion
  const tableColumnsString = tableColumns.map(column => `${column.name} ${column.dataType} ${getEncodingColumnString(column)}`).join();
```

Reply:
We could down this route, but it'll change the outputting text for a lot of strings, that append to this string. I've noticed that almost every test in the table test file needs to be changed to accommodate this change.

Reply:
This has been done